linux/open_vm_tools/add_enable_service.yml

```diff
@@ -56,11 +56,19 @@
         service_enabled: true
         service_state: "started"
 
-- name: "Enable FreeBSD service {{ service_name }}"
+- name: "Update tools.conf path and enable FreeBSD service {{ service_name }}"
   when:
     - guest_os_family == 'FreeBSD'
     - service_name == 'vmware-guestd'
   block:
+    - name: "Update tools.conf path to /etc/vmware-tools/tools.conf when tools is installed from source code"
+      ansible.builtin.replace:
+        path: "{{ service_file }}"
+        regexp: '/usr/local/share/vmware-tools/tools.conf'
```

Comment:
Do not hard code the tools.conf path. Please use a regex pattern like: `-c .*tools.conf`